_codux/ui-kit/components/components.board.tsx

```diff
@@ -17,46 +16,50 @@ export default createBoard({
                     <span className={styles.foundation}> | Core components</span>
                     <hr className={styles.hrSolid} />
                     <h3 className={styles.sectionTitle}>Components &amp; Elements</h3>
-                    <h4 className={styles.sectionHeader}>INPUT</h4>
                 </div>
-                <QuantityInput value={6} className={styles.quantityInput1} onChange={() => {}} />
-                <span className={styles.fontDetails}>Number Input</span>
-                <Variant name="Heading1">
-                    <hr className={styles.hrLight} />
-                    <h4 className={styles.sectionHeader}>ACCORDION</h4>
-                    <Accordion
-                        items={[
-                            {
-                                title: 'Product Info',
-                                content: 'Content',
-                            },
-                            {
-                                title: 'Return & Refund Policy',
-                                content: 'Content',
-                            },
-                            {
-                                title: 'Shipping Info ',
-                                content: 'Content',
-                            },
-                        ]}
-                        className={styles.accordion}
-                    />
-                    <hr className={styles.hrLight} />
+                <h4 className={styles.sectionHeader}>INPUT</h4>
+                <Variant name="Number Input">
+                    <QuantityInput value={6} onChange={() => {}} />
                 </Variant>
-                <p className={classNames(styles.variantName, styles.headlinesSpacing)}></p>
+                <span className={styles.variantName}>Number Input</span>
+                <hr className={styles.hrLight} />
+                <h4 className={styles.sectionHeader}>ACCORDION</h4>
+                <div style={{ width: '70%' }}>
+                    <Variant name="Accordion">
```

Comment:
Why do we wrap components into `Variant` if we have them in `AddPanel` ?

Reply:
No idea 🤷‍♂️ Doesn't make sense to me as it creates duplicate instances of the same thing in the panel. But this is what they want in UI Kits.

Reply:
This PR is just a cleaned-up version of https://github.com/codux-templates/e-commerce-reclaim/pull/105